### PR TITLE
Update readme&setup with PG config

### PR DESCRIPTION
**Pull request by smialko**
Updated instructions for postgres setup

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,36 @@ yarn dev
 rails s
 ```
 
+## Postgres setup
+
+```bash
+brew services start postgresql
```
Comment (franiakat) [inline README.md:49]: First we need to install it. Most cases one will have it installed from other projects, but still.
`brew install postgresql`

File: README.md [hunk 2]
```diff
@@ -43,6 +43,36 @@ yarn dev
...
+```
+
+# In the database.yml change development configuration to:
+
+development:
+adapter: postgresql
+database: aircasting_development
+username: root
+password:
```
Comment (franiakat) [inline README.md:58]: This is included in database.yml.example. One must make sure they have the file copied and renamed to database.yml.

Comment (franiakat) [reply README.md:58]: Oh, I see, you meant the username change. Maybe we can just include that in example?

File: README.md [hunk 3]
```diff
@@ -43,6 +43,36 @@ yarn dev
...
+psql -d aircasting_development
+CREATE ROLE postgres WITH CREATEDB CREATEROLE LOGIN;
+```
+
+# In the database.yml change development configuration to:
+
+development:
+adapter: postgresql
+database: aircasting_development
+username: postgres
```
Comment (franiakat) [inline README.md:73]: And then here refer to the username change as well.

File: README.md [hunk 4]
```diff
@@ -43,6 +43,36 @@ yarn dev
 rails s
 ```
 
+## Postgres setup
```
Comment (franiakat) [inline README.md:46]: @BartkiewiczM Please take a look, this is what we did locally to make the app work with PG. But I believe you've been in touch with PG way more and may have some suggestions to simplify things.

**Review by jankrzempek — COMMENTED** [review]
Nothing to add from me, I do not feel reliable in this

File: README.md (hunk 4 above)
Comment (smialko) [reply README.md:46]: @BartkiewiczM Are you testing or planning to test in a near future? 
@franiakat maybe we could add it as it is - any changes for simplification of this process can be applied without much work, and they won't influence anything as it's only the readme :)

Comment (BartkiewiczM) [reply README.md:46]: I've realized that all the people that had mysql setup already switched to postgres. I think while setting up from scratch there will be no/different problems. I'll test a clean setup on a new machine this week and come back here :)

**Comment by BartkiewiczM** [comment]
I've edited the setup file and readme. I've tested it once on a new mac and it was fine, tomorrow I'll test it on a currently used mac and it will be ready to merge.

**Review by mpolec — APPROVED** [review]
🙌

**Pull request merged**